pkg/arvo/sys/vane/ames.hoon

```diff
@@ -1032,26 +1069,37 @@
   [moves ames-gate]
 ::  +stay: extract state before reload
 ::
-++  stay  [%2 %adult ames-state]
+++  stay  [%3 %adult ames-state]
 ::  +load: load in old state after reload
 ::
 ++  load
   |=  $=  old-state
       $%  [%1 ames-state-1]
-          [%2 ^ames-state]
+          [%2 ames-state-2]
+          [%3 ^ames-state]
       ==
   ^+  ames-gate
   ?-    -.old-state
-      %2
+      %3
     ames-gate(ames-state +.old-state)
+  ::
```

Comment:
This looks trivial enough to be fine, but I generally upgrade one at a time with a sequence of `=?`s, else it can't be tested without trying upgrades from each of the old versions -- a quadratic problem.  Example:

https://github.com/urbit/urbit/blob/0c95c2d4dd3a7d1caccccd1412d0b2fdec104d49/pkg/arvo/sys/vane/gall.hoon#L1554-L1566